source/codegen/metadata_mutation.py

```diff
@@ -85,12 +87,30 @@ def mark_attr_value_param_if_required(function, enums, attribute_enums_by_type,
         enum_type = attribute_value_param["type"]
     if attribute_value_param != None and enum_type in attribute_enums_by_type:
         enum_name = get_attribute_values_enum_name(service_class_prefix, enum_type)
-        if(enum_name in enums):
-            attribute_value_param['enum'] = enum_name
+        if enum_name in enums or f'{enum_name}Mapped' in enums:
+            if enum_name in enums:
+                attribute_value_param['enum'] = enum_name
+            if f'{enum_name}Mapped' in enums:
+                attribute_value_param['mapped-enum'] = f'{enum_name}Mapped'
         else:
             # Ideally there must be an enum associated with this parameter for this SetAttribute* API.
             # Since the enum is empty, users will be passing in raw values. Make it clear via naming.
             attribute_value_param['name'] = attribute_value_param['name'] + "_raw"
```

Comment:
This place adds `_raw` suffix to "value" params of `SetAttribute` APIs where there is no associated enum. As the comment above mentions, it keeps all `SetAttribute` APIs consistent even if some of them don't have enum for the `value` parameter.  For e.g. [this API](https://github.com/ni/grpc-device/blob/main/generated/nidcpower/nidcpower.proto#L1844) in dcpower

If we don't rename the parameter here, since there is no enum associated with it, codegen will generate `request` message with `value` as parameter name - and this parameter actually accepts raw value unlike similar APIs from other drivers where `value` parameter accepts enum. This would be confusing for users.

Reply:
How is that confusing to customers? If there's a field that looks like this, it's clear that it's raw and how to use it.
```
double value = 5;
```

Reply:
Yes, it is clear when you look at proto file. But when you are using it as API, it would be confusing if you have to use `.value` for setting raw value in SetAttributeViInt64Request  of [nidigitalpattern](https://github.com/ni/grpc-device/blob/main/generated/nidigitalpattern/nidigitalpattern.proto#L1687), but have to use `.value_raw` for setting raw value in same API of [niscope](https://github.com/ni/grpc-device/blob/main/generated/niscope/niscope.proto#L1821). The linked files are with this change where I rename `value` to `value_raw`, but imagine the digital pattern file with param name as `.value` above.